src/Storefront/Resources/views/storefront/element/cms-element-image-gallery.html.twig

```diff
@@ -347,7 +347,7 @@
                                 {% endblock %}
 
                                 {% block element_image_gallery_slider_dots %}
-                                    {% if imageCount > 1 and navigationDots %}
+                                    {% if imageCount > 1 and navigationDots != 'Keine' %}
```

Comment:
does this work for every admin language? :thinking:

Reply:
@mitelg as I mentioned above in my comment. I don't know if the value in another languages is also "Keine" if no, then we need another generic solution here. For example "none" like with the "outside" setting

Currently I’m updating German only installation and this bug occurred.

Reply:
sorry, I missed that :see_no_evil: 

I can't believe that "Keine" is hard coded :grimacing: 

would be nice, if you could check that

Reply:
@mitelg Created new language on my instance. The value is "Keine" also in english.

Reply:
my shop was installed with English and if I use English as admin language, the value is "None" on my side :grimacing: 

so it seems, that the label value is saved in the database, which is of course unfortunate. 

so this needs to be fixed in a language agnostic way

Reply:
@mitelg Okay. Then you can deny my pull request and maybe create another issue for this. Thanks for the check, i will temporary override the template in my theme, until its fixed

Reply:
would be nice, if you could do that :blush: